### ISSUE TITLE
Loss not improving

### ISSUE THREAD

**Issue by EmileTestUser** [david8862/keras-YOLOv3-model-set#126]
Hi there,

when I try to run vanilla repo code and train it on the COCO dataset the loss does not continue to improve at some point. From around epoch 15 loss will stay around 70.

I have standard settings. I used the repo tools to convert the dataset.

I have tried to increase tranfer-epochs and/or reduce the learning rate but its still the same issue.
Any Ideas on this?

**Comment by david8862** [comment]
@EmileTestUser  which model type do you use?

**Comment by EmileTestUser** [comment]
I am using yolo4_darknet.
![Capture](https://user-images.githubusercontent.com/31615481/98466588-c364a800-21d0-11eb-8fea-357af1e3a1ac.PNG)

**Comment by david8862** [comment]
@EmileTestUser for darknet backbone families, now this repo couldn't achieve a good training performance due to lack of valid Imagenet pretained backbone weights. Maybe you can try some other backbone type. Sorry for trouble
